sssom/util.py

```diff
@@ -636,7 +638,10 @@ def deal_with_negation(df: pd.DataFrame) -> pd.DataFrame:
                 (combined_normalized_subset[SUBJECT_ID] == row_1[SUBJECT_ID])
                 & (combined_normalized_subset[OBJECT_ID] == row_1[OBJECT_ID])
                 & (combined_normalized_subset[CONFIDENCE] == row_1[CONFIDENCE])
-                & (combined_normalized_subset[MATCH_TYPE] == HUMAN_CURATED_MATCH_TYPE)
+                & (
+                    combined_normalized_subset[MAPPING_JUSTIFICATION]
+                    == "semapv:ManualMappingCuration"
```

Comment:
Better use a token here again rather than a string, maybe something like `SEMPAV.ManualMappingCuration` in the SEMAPV enum in constants?